src/Bitcoin/Entry.cpp

```diff
@@ -23,20 +22,29 @@ bool Entry::validateAddress(TWCoinType coin, const std::string& address, byte p2
     case TWCoinTypeQtum:
     case TWCoinTypeViacoin:
     case TWCoinTypeBitcoinGold:
-        return SegwitAddress::isValid(address, hrp) || Address::isValid(address, {{p2pkh}, {p2sh}});
-
+        if (auto* prefix = std::get_if<UTXOPrefix>(&addressPrefix); prefix) {
```

Comment:
I think `UTXOPrefix` is not the best name, it is an address type / address prefix, not specific to UTXOs.

Reply:
> I think `UTXOPrefix` is not the best name, it is an address type / address prefix, not specific to UTXOs.

Trying to find an appropriate name here, I've checked the registry.json and only UTXO coins use p2sh/p2pkh. Which name we can use instead?